_drafts/EH-tidyx.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Community Highlight: Ellis Hughes's TidyX Screencast!"
+author: Lauren Wolfe
+categories:
+  - Community
+  - Technical
+tags: 
+  - highlight
+  - tidyverse
+  - R
+---
+
+As we've been reassessing our work here at The Coop, we have found that there is a need to facilitate more interaction between the people that make up our community. It makes sense - we no longer have easy access to a quick catch up in the hallways or over lunch on campus and many of us have settled into our new socially-distant lifestyle. This is why The Coop Blog is going to shift some focus back onto our community by highlighting members and their data related work. For our first community highlight we're showcasing Ellis and his TidyX screencast!
+
+Ellis is a self-described bioengineer turned data scientist who works as a statistical programmer within the Statistical Center for HIV/AIDS Research and Prevention (SCHARP) here at Fred Hutch. Folks who've spent a lot of time around The Coop or the R community in general will likely already be familiar with him as he's an regular user of The Coop Slack channel and has hosted various Coop community groups such as the data visualizaion group. Outside of his work at Fred Hutch, Ellis is an active member of the larger R community contributing to open-source projects and organizing the local Seattle UseR group.
+
+To keep his skills sharp Ellis regularly takes part in Tidy Tuesday, a project out of the R for Data Science Online Learning Community. The goal of Tidy Tuesday is to provide a constructivce and supportive space for new and seasoned R users alike to practice their data wrangling and visualization skills. The premise is that each week a dataset is shared out via the [TidyTuesday github repository](https://github.com/rfordatascience/tidytuesday). The dataset has generally been "tamed", meaning that some cleaning has been done, but it doesn't meet the standards of being "tidy". [Tidy data](https://cran.r-project.org/web/packages/tidyr/vignettes/tidy-data.html#:~:text=Tidy%20data%20is%20a%20standard,Each%20observation%20forms%20a%20row.) is a term coined by Hadley Wickham. The principles behind tidy data act to provide a standard way of organizing data values in a dataset to facilitate analysis. Participants are asked to use techniques from the (R for Data Science book)[https://r4ds.had.co.nz/] to explore and tidy the dataset and finally create a visualization to share out to the R statistics community via Twitter using the hashtag [#TidyTuesday](https://twitter.com/search?q=%23TidyTuesday&src=typeahead_click).
```

Comment:
is the tidy data link supposed to go to a certain part of the page? Might be easier to truncate it to https://cran.r-project.org/web/packages/tidyr/vignettes/tidy-data.html to avoid broken links later